INSTALLATION.md

```diff
@@ -62,4 +64,23 @@ Congratulations! You have successfully installed **MassaStation** on your comput
 6. The installation will begin, and you will see progress information in the terminal.
 7. Once the installation is complete, you can close the terminal.
 
-Congratulations! You have successfully installed **MassaStation** on your computer. You can start MassaStation by searching for it in the Applications folder.
+Congratulations! You have successfully installed **MassaStation** on your computer. You can start **MassaStation** by searching for it in the Applications folder.
+
+
+## Uninstallation
+
+### MacOS
+
+To uninstall **MassaStation** from your MacOS system, follow the steps below:
+
+1. Open the Terminal application on your MacOS system.
+2. Execute the following command in the terminal to download and run the **MassaStation** uninstaller script:
```

Comment:
I think it would be cleaner to isntaller the uninstaller at the same time as the massa station app is installed.
Following this approach you will be sure that the installer is compatible with the version of Massa Station and you will have nothing to install at this step.